src/datastore/pg-write-store.ts

```diff
@@ -715,6 +725,45 @@ export class PgWriteStore extends PgStore {
     }
   }
 
+  async fixBlockZeroData(sql: PgSqlClient, blockOne: DbBlock): Promise<void> {
+    const tablesUpdates: Record<string, number> = {};
+    const txsResult = await sql<TxQueryResult[]>`
+      UPDATE txs
+      SET 
+        canonical = true,
+        block_height = 1,
+        tx_index = tx_index + 1,
```

Comment:
Shouldn't this be `last_block_1_tx_index + 1`? IIRC block 1 has like 2 or 3 transactions within it

Reply:
In theory it seems like there could have been an alternative history where block 1 also included some arbitrary additional txs, but that didn't happen in current mainnet so we don't have to worry about it.